package.json

```diff
@@ -71,6 +71,18 @@
     }
   },
   "dependencies": {
+    "@emotion/react": "^11.11.4",
+    "@emotion/styled": "^11.11.0",
+    "@hookform/resolvers": "^3.3.4",
+    "@mui/material": "^5.15.14",
+    "@patternfly/react-table": "^5.2.4",
+    "@perses-dev/components": "0.0.0-snapshot-scatterplot-fix-imports-95e1b59",
```

Comment:
Do we still need a snapshot or can we point to the latest perses release

Reply:
We still need the snapshot; I had to create another PR last week to fix the scatterplot imports. This is beyond the current v.44 release.

Reply:
[OU-385](https://issues.redhat.com/browse/OU-385)